### ISSUE TITLE
Add a .gitattributes file to enforce line endings

### ISSUE THREAD

**Issue by sofiageo** [grommet/grommet#3437]
### Expected Behavior
People using Windows and Linux can commit with the same line endings (LF)

### Actual Behavior
Windows users with mis-configured git clients commit files with CRLF line endings.

### URL, screen shot, or Codepen exhibiting the issue
Example:
https://github.com/grommet/grommet/issues/3394#issuecomment-539166085

### Steps to Reproduce
Clone grommet from a Windows PC, then try to commit some changes.

### Your Environment
- Operating System and version (desktop or mobile): Windows 

### Solution
A simple .gitattributes file that enforce line endings for text files. Github also has an article about it: https://help.github.com/en/articles/configuring-git-to-handle-line-endings

**Comment by sofiageo** [comment]
I'm adding some examples from random famous libraries / software: 

1) https://chromium.googlesource.com/v8/v8/+/refs/heads/master/.gitattributes
2) https://github.com/mozilla/pdf.js/blob/master/.gitattributes
3) https://github.com/styled-components/styled-components/blob/master/.gitattributes
4) https://github.com/microsoft/TypeScript/blob/master/.gitattributes
5) https://github.com/facebook/react/blob/master/.gitattributes

My experience is with examples 1,2 and git configuration: core.autocrlf=input

**Comment by ShimiSun** [comment]
@oorestisime curious to hear your thoughts on this one.

**Comment by oorestisime** [comment]
I agree on adding one. i am not sure which one is better and for what reason. lets go with react?

**Comment by abhinandanas** [comment]
Hi! I'd like to work on this issue. I'll add a .gitattributes file to enforce consistent LF line endings across the repo. Please assign it to me. 🙏
